client/components/marketing-survey/cancel-auto-renewal-form/index.jsx

```diff
@@ -0,0 +1,148 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize, moment } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Dialog from 'components/dialog';
+import FormSectionHeading from 'components/forms/form-section-heading';
+import FormButton from 'components/forms/form-button';
+import FormButtonsBar from 'components/forms/form-buttons-bar';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+import FormTextInput from 'components/forms/form-text-input';
+import { submitSurvey } from 'lib/upgrades/actions';
+import enrichedSurveyData from 'components/marketing-survey/cancel-purchase-form/enriched-survey-data';
+import './style.scss';
+
+const OTHER_FEEDBACK = 'other-feedback';
+
+class CancelAutoRenewalForm extends Component {
+	static propTypes = {
+		purchase: PropTypes.object.isRequired,
+		selectedSite: PropTypes.object.isRequired,
+		isVisible: PropTypes.bool,
+		onClose: PropTypes.func.isRequired,
+		translate: PropTypes.func.isRequired,
+	};
+
+	state = {
+		response: '',
+		feedback: '',
+	};
+
+	radioButtons = {};
+
+	constructor( props ) {
+		super( props );
+
+		const { translate } = props;
+
+		this.radioButtons = [
+			[ 'take-a-break', translate( "Yes, I'm just taking a break for now." ) ],
```

Comment:
Out of curiosity, why "I'm" and "don’t" use different single quotes?

Reply:
Nice catch! It was simply a mistake from copying & pasting. Updated in a8fe4d0.